README.md

```diff
@@ -6,7 +6,7 @@ GOV.UK stack.
 ## How to use
 
 ```shell
-rails new APP_NAME -m http://raw.github.com/govuk-rails-app-template/master/template.rb
+rails new APP_NAME --skip-test-unit --skip-spring -m govuk-rails-app-template/template.rb
```

Comment:
It's also worth prefixing this command with `RBENV_VERSION=2.2.2`, otherwise it'll run under the default ruby, which is typically the system ruby.